test/testdata/wrapper/lts-11.14/hie.yaml

```diff
@@ -1,3 +0,0 @@
-# TODO: generate this in test suite
-cradle:
-  stack:
```

Comment:
Are you generating these in the test-suite now? otherwise, they are needed.

Reply:
Moved to `test/testdata/wrapper/lts-14.18/hie.yaml`